toc/rfc/rfc-draft-repository-ownership.md

```diff
@@ -0,0 +1,78 @@
+
+The subset of unmanaged active repositories SHOULD be empty under steady-state operation. The community MUST collectively act to resolve the status of an unmanaged active repository within a reasonable period of time, either by archiving it or by placing it under the ownership of a  Working Group, of the TOC, or of the CFF staff, as is appropriate.
+
+The ownership structure of the repositories in the `cloudfoundry-incubator` GitHub organization can be described the same way as a partition of the set of repositories amongst the Working Groups. Since incubation is no longer a relevant concept in CF Foundation governance, the community SHOULD act to move all of the active repositories in this organization to the main `cloudfoundry` GitHub organization in a reasonable period of time, while coordinating with the Working Groups to minimize negative effects on development and release practices. The community MUST NOT create new repositories in the `cloudfoundry-incubator` GitHub organization and MUST NOT move any existing repositories into it.
+
+
+### Ownership changes
+
+A change to repository ownership is a change to the partition of the set of repositories in the `cloudfoundry` organization. For a single repository, the possible changes are the addition of a new repository to the one of the subsets in the partition, the transfer of an existing repository from one subset to another, or the removal of a repository from a subset.
+
+Addition of a repository to the partition corresponds either to creating a new repository or transferring an existing repository from another GitHub organization.
+
+Removal of a repository corresponds either to transferring it to another GitHub organization or to deleting the repository entirely, neither of which should be undertaken lightly. The CF community SHOULD ensure that either of these cases receives thorough consideration and scrutiny before proceeding.
+
+
+### Ownership change approvals
+
+The TOC MUST approve any change to repository ownership, just as it originally approved the initial set of repositories for each Working Group. Approval consists of a quorum decision by the TOC.
+
+Additionally, if a change affects the subset of repositories that a Working Group owns, that Working Group MUST approve the change as well. Approval consists of a quorum decision by the leads of the Working Group.
+
+The TOC MUST also approve of any proposal to create a new repository within the `cloudfoundry` GitHub organization or to rename an existing repository. 
+
+Proposed changes to repository ownership SHOULD be submitted for consideration as a pull request or issue on the [Cloud Foundry community repository](https://github.com/cloudfoundry/community) to provide visibility in the CF community, to encourage asynchronous discussion, and to reduce fragmentation of discussion.
```

Comment:
Do we need issues or can we use PRs only?
I would add that the PR should modify the WG charter documents (= not the technical cloudfoundry.yml which is eventually generated from the charters).

Proposed changes to repository ownership SHOULD be submitted for consideration as a pull request on the Working Group charter documents in the Cloud Foundry community repository ...

Reply:
Looks good now, thanks.